samples/word/50-document/manage-styles.yaml

```diff
@@ -0,0 +1,332 @@
+            style.load();
+            await context.sync();
+
+            if (style.nameLocal) {
+              console.warn(
+                `There's an existing style with the same name '${newStyleName}'! Please provide another style name.`
+              );
+              return;
+            }
+
+            const newStyleType = ($("#new-style-type").val() as unknown) as Word.StyleType;
+            context.document.addStyle(newStyleName, newStyleType);
+            console.log(newStyleName + " has been added to the style list.");
+          });
+        }
+
+        async function getProperties() {
+          // Get the properties of the specified style.
+          await Word.run(async (context) => {
+            const styleName = $("#style-name-to-use").val() as string;
+            if (styleName == "") {
+              console.warn("Enter a style name to get properties.");
+              return;
+            }
+
+            const style = context.document.getStyles().getByNameOrNullObject(styleName);
+            style.load();
+            await context.sync();
+
+            if (!style.nameLocal) {
+              console.warn(`There's no existing style with the name '${styleName}'.`);
+            } else {
+              style.font.load();
+              style.paragraphFormat.load();
+              await context.sync();
+
+              console.log(`Properties of the '${styleName}' style:`);
+              console.log(style);
+            }
+          });
+        }
+
+        async function applyStyle() {
+          // Apply the specified style to a paragraph.
+          await Word.run(async (context) => {
+            const styleName = $("#style-name-to-use").val() as string;
+            if (styleName == "") {
+              console.warn("Enter a style name to apply.");
+              return;
+            }
+
+            const style = context.document.getStyles().getByNameOrNullObject(styleName);
+            style.load();
+            await context.sync();
+
+            if (!style.nameLocal) {
+              console.warn(`There's no existing style with the name '${styleName}'.`);
+            } else if (style.type != Word.StyleType.paragraph) {
+              console.log(`The '${styleName}' style isn't a paragraph style.`);
+            } else {
+              const body = context.document.body;
+              body.clear();
+              body.insertParagraph(
+                "Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.",
+                "Start"
+              );
+              const paragraph = body.paragraphs.getFirst();
+              paragraph.style = style.nameLocal;
+              console.log(`'${styleName}' style applied to first paragraph.`);
+            }
+          });
+        }
+
+        async function setFontColor() {
```

Comment:
the cases will grow more and more, I think this SetFontColor and the following SetFontSize can be merged as one for simplity